README.md

```diff
@@ -102,14 +102,12 @@ workflow2 = Floe::Workflow.load("workflow2.asl")
 
 running_workflows = [workflow1, workflow2]
 until running_workflows.empty?
-  # Wait for any of the running workflows to be ready (up to the timeout)
-  ready_workflows = Floe::Workflow.wait(running_workflows)
```

Comment:
The point of this example was to show how to use `Floe::Workflow.wait([])`.  This looks more like the example code above this but with more than one workflow.

That's fine but we still need to keep the `Floe::Workflow.wait([])` example.

Also I think this code would fail since `ready_workflows` is undefined.